generated/installer/ibm-spectrum-scale-csi-operator-ocp-rhel.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: ibm-spectrum-scale-csi-operator
       containers:
       - name: operator
-        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator@sha256:c292fa07a03f5accac554a5f366b7241c31dbb85fc4617c0fe2428cf103fce56
```

Comment:
I think we should not deliver this yaml as a part of 2.13 and beyond.

Reply:
Deleted the manifests for OCP+RHEL